dawn/src/dawn/Support/StringUtil.h

```diff
@@ -129,4 +129,16 @@ extern std::string decimalToOrdinal(int dec);
 
 /// @}
 
+/// @brief Checks if two strings are equal (case insensitive)
+/// @ingroup support
+bool equals_lower(const std::string& a, const std::string& b);
```

Comment:
these should probably be camelCase for consistency w/ rest of codebase

Reply:
I thought that we wanted standalone methods to use snake case. @twicki?

Reply:
@twicki and I discussed this and landed on camelCase, so I changed it to use that.